velox/functions/prestosql/ArraySort.cpp

```diff
@@ -374,14 +374,14 @@ std::vector<std::shared_ptr<exec::FunctionSignature>> signatures(
   std::vector<std::shared_ptr<exec::FunctionSignature>> signatures = {
       // array(T) -> array(T)
       exec::FunctionSignatureBuilder()
-          .typeVariable("T")
+          .orderableTypeVariable("T")
```

Comment:
Consider updating the documentation to add this restriction.

Reply:
I've updated all the places related to array_sort in the document.